test/arrays/vass/const-checking.aliases.good

```diff
@@ -1,9 +1,2 @@
-const-checking.aliases.chpl:8: In function 'main':
-const-checking.aliases.chpl:10: error: illegal lvalue in assignment
-const-checking.aliases.chpl:12: error: illegal lvalue in assignment
-const-checking.aliases.chpl:14: error: creating a non-const alias 'aa2' of a const array or domain
-const-checking.aliases.chpl:18: error: creating a non-const alias 'dd2' of a const array or domain
-const-checking.aliases.chpl:23: error: illegal lvalue in assignment
-const-checking.aliases.chpl:24: error: illegal lvalue in assignment
-const-checking.aliases.chpl:27: error: illegal lvalue in assignment
-const-checking.aliases.chpl:28: error: illegal lvalue in assignment
```

Comment:
If you turn the bad 'ref' into a 'const ref', you should then see the "illegal lvalue in assignment" failures (I will verify this on my end, but I'm pretty sure I tried this before).